include/network-config-manager.h.in

```diff
@@ -114,6 +114,8 @@ int ncm_configure_link_coalesce(int argc, char *argv[]);
 int ncm_configure_link_coald_frames(int argc, char *argv[]);
 int ncm_configure_link_coal_pkt(int argc, char *argv[]);
 
+int ncm_configure_add_sr_iov(int argc, char *argv[]);
+
```

Comment:
`ncm_configure_add_sr_iov` -> `ncm_configure_sr_iov`

Reply:
Thanks @ssahani for review, And all the review comments has been taken care.